marlowe-cli/test/double-satisfaction.sh

```diff
@@ -71,11 +76,11 @@ cardano-cli query utxo --testnet-magic "$MAGIC" --address "$PARTY_A_ADDRESS"
 # We select the UTxO with sufficient funds to use in executing the contract.
 
 TX_0_PARTY_A=$(
-marlowe-cli util select --testnet-magic "$MAGIC"                  \
-                        --socket-path "$CARDANO_NODE_SOCKET_PATH" \
-                        --lovelace-only 50000000                  \
-                        "$PARTY_A_ADDRESS"                        \
-| sed -n -e '1{s/^TxIn "\(.*\)" (TxIx \(.*\))$/\1#\2/;p}'         \
+$MARLOWE_CLI_BIN -- util select --testnet-magic "$MAGIC"                \
+                        --socket-path "$CARDANO_NODE_SOCKET_PATH"       \
+                        --lovelace-only 50000000                        \
+                        "$PARTY_A_ADDRESS"                              \
+| tail -n +2 | sed -n -e '1{s/^TxIn "\(.*\)" (TxIx \(.*\))$/\1#\2/;p}'  \
```

Comment:
Why does this skip two lines of UTxOs (via `tail -n +2`)? Wouldn't this only work if there are at least three UTxOs at the address? (Also, the `tail -n +2` would have been redundant anyway, since the `sed` could just select the third line if that is what we want.)

Reply:
I was trying to make it work with both `marlowe-cli` or `cabal run marlowe-cli --` but this is broken for `marlowe-cli` and probably too complicated to bother with. I'll get rid of that and we can use rely on having the binary built with nix.